tests/utility/hash_test.cu

```diff
@@ -39,12 +39,13 @@ struct large_key {
   int32_t data_[Words];
 };
 
-template <typename Hash, typename Seed = typename Hash::result_type>
-__host__ __device__ bool check_hash_result(typename Hash::argument_type const& key,
-                                           Seed seed,
-                                           typename Hash::result_type expected) noexcept
+template <typename Hash, typename... HashConstructorArgs>
+static __host__ __device__ bool check_hash_result(
+  typename Hash::argument_type const& key,
+  typename Hash::result_type expected,
+  HashConstructorArgs&&... hash_constructor_args) noexcept
 {
-  Hash h(seed);
+  Hash h(std::forward<HashConstructorArgs>(hash_constructor_args)...);
```

Comment:
```suggestion
  Hash h(cuda::std::forward<HashConstructorArgs>(hash_constructor_args)...);
```
With the support of CCCL, we probably want to use `cuda::std::forward` since this is a host-device function.

Rules of thumb for using STL utilities:

- `std::` in host function
- `cuda::` for device function
- `cuda::std::` for host-device function

Reply:
Thanks for the info!